modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx

```diff
@@ -425,6 +425,26 @@ See below:
         licorice100
       </Table.Cell>
     </Table.Row>
+    <Table.Row>
+      <Table.Cell>Switch</Table.Cell>
+      <Table.Cell />
+      <Table.Cell>disabled</Table.Cell>
+      <Table.Cell>opacity</Table.Cell>
+      <Table.Cell backgroundColor="frenchVanilla100">1</Table.Cell>
+      <Table.Cell backgroundColor="frenchVanilla100">
```

Comment:
Can you remove the Pill stuff, there were some changes to it in my RP, I'll update it in my PR!

Reply:
Sorry i mean Switch

Reply:
Will do!